src/shared_modules/utils/windowsHelper.h

```diff
@@ -175,6 +192,52 @@ namespace Utils
         return ret;
     }
 
+    static std::string normalizeTimestamp(const std::string& timestamp1, const std::string& timestamp2)
```

Comment:
timestamp1 is expected to be a date. Change the name.

Reply:
Changed to a more descriptive name for those two variables.